src/trait-bounds.md

```diff
@@ -156,6 +156,35 @@ fn call_on_ref_zero<F>(f: F) where F: for<'a> Fn(&'a i32) {
 }
 ```
 
+## Implied bounds
+
+Rust sometimes infers some bounds the user would have otherwise been required to write.
+
+```rust
+fn requires_t_outlives_a<'a, T>(x: &'a T) {}
+```
+While this function requires `t` to outlive `'a`, this is inferred as the function signature
```

Comment:
Should `t` here be `T`?

Reply:
Also,  I think that `inferred because the function` is a bit easier to read than `inferred as the function`.